grunt-config-options.js

```diff
@@ -50,6 +50,15 @@ module.exports = {
     connect: {
         server: {
             options: {
+                middleware: function(connect, options, middlewares) {
+                  middlewares.push(
+                    connect.use(function(request, response, next) {
+                      response.setHeader('Access-Control-Allow-Origin', 'http://crossorigin.test');
```

Comment:
I think that this may be logically inverted; the origin to allow loading resources from is the `connect` webserver on port 4000, not the domain from which the resources are being loaded.

I have some commits staged that I'll push in a few moments that improve the naming of the test webserver hostnames in this branch and also that adjust this header configuration.

Reply:
...except that the relevant response would be sent by the remote server, and in the case of the tests, those are mocked out.  So: I think we can remove this as unnecessary.